angr/procedures/win32/gethostbyname.py

```diff
@@ -0,0 +1,13 @@
+import angr
+import logging
+import claripy
+
+l = logging.getLogger('angr.procedures.win32.gethostbyname')
+
+class gethostbyname(angr.SimProcedure):
+
+    def run(self, _): #pylint:disable=arguments-differ
+        self.argument_types = {0: self.ty_ptr(angr.sim_type.SimTypeString()), }
+        self.return_type = self.ty_ptr(angr.sim_type.SimTypeTop(self.state.arch))
```

Comment:
argument_types and return_type don't do anything and have never done anything. If you want to contribute the types to a database you should put them in the SimLibrary.

Reply:
Oh, I inherited this from previous work and didn't bother to check, my bad, I'll remove them.